VirtoCommerce.InventoryModule.Data/Services/InventorySearchService.cs

```diff
@@ -40,15 +94,76 @@ public virtual GenericSearchResult<InventoryInfo> SearchInventories(InventorySea
                 query = query.OrderBySortInfos(sortInfos).ThenBy(x => x.Id);
 
                 result.Results = query.Skip(criteria.Skip)
-                                 .Take(criteria.Take)
-                                 .AsEnumerable()
-                                 .Select(x => x.ToModel(AbstractTypeFactory<InventoryInfo>.TryCreateInstance()))
-                                 .ToList();
+                    .Take(criteria.Take)
+                    .AsEnumerable()
+                    .Select(x => x.ToModel(AbstractTypeFactory<InventoryInfo>.TryCreateInstance()))
+                    .ToList();
+
             }
 
             return result;
         }
 
+        /// <summary>
+        /// Build column name map from resulting FulfillmentCenterInventoryInfo to FulfillmentCenterInventory object fields by which we queue the data
+        /// </summary>
+        private void InitSortingAliases()
```

Comment:
Let's make it `protected virtual` too, as it may require an extension in case of custom fields.